spec/rails_helper.rb

```diff
@@ -3,6 +3,8 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'shoulda/matchers'
+require 'sidekiq/testing'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.